### DISCUSSION TITLE
Zigbee Tasmota Status 99

### DISCUSSION THREAD

**Discussion by Mordi1984** [arendst/Tasmota#10928] [Q&A] [answered]
Hi,

I really dont know, but my Zigbee device drives me crazy!
Sunday, First time Status 99.
Well i pushed after all accidently the auto update and then 9.2. No Upgrade to Sbbride possible after all.
I seeked me FTDI, installed because of an crappy old device an emulated Windows XP.
Fighted wit Cables, and reflashed it.
After a long time i received the message, Zigbee started.
Today i plugged the device out, and got him energy from an Fritzbox USB Port.
Then after 2 restarts again Status 99!
Why!!!!!!!
The device didnt fall on the ground.
If i restore Datas, still Status 99!
What is wrong!
I really dont know and it become insane.

LOG:
00:00:00.529 WIF: Connecting to AP1 Endlich Zuhause Channel 1 BSSId C8:0E:14:A0:46:B1 in mode 11n as tasmota_E32AEB-2795...
00:00:01.752 WIF: Connected
00:00:02.002 HTP: Web server active on tasmota_E32AEB-2795 with IP address 192.168.178.131
00:19:43.002 MQT: Attempting connection...
00:19:43.023 MQT: Connected
00:19:43.025 MQT: tele/tasmota_E32AEB/LWT = Online (retained)
00:19:43.026 MQT: cmnd/tasmota_E32AEB/POWER = 
00:19:43.034 MQT: tele/tasmota_E32AEB/INFO1 = {"Module":"Sonoff ZbBridge","Version":"9.2.0.6(zbbridge)","FallbackTopic":"cmnd/DVES_E32AEB_fb/","GroupTopic":"cmnd/tasmotas/"}
00:19:43.036 MQT: tele/tasmota_E32AEB/INFO2 = {"WebServerMode":"Admin","Hostname":"tasmota_E32AEB-2795","IPAddress":"192.168.178.131"}
00:19:43.039 MQT: tele/tasmota_E32AEB/INFO3 = {"RestartReason":"Software/System restart"}
00:19:46.489 QPC: Reset
00:19:47.458 MQT: tele/tasmota_E32AEB/STATE = {"Time":"2021-02-12T00:19:47","Uptime":"0T00:00:09","UptimeSec":9,"Vcc":3.475,"Heap":31.4,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":1,"Wifi":{"AP":1,"SSId":"Endlich Zuhause","BSSId":"C8:0E:14:A0:46:B1","Channel":1,"RSSI":100,"Signal":-23,"LinkCount":1,"Downtime":"0T00:00:03"}}
00:19:55.033 ZIG: Resetting EZSP device
00:19:56.310 MQT: tele/tasmota_E32AEB/RESULT = {"ZbState":{"Status":1,"Message":"EFR32 booted","RestartReason":"Power-on","Code":2}}
00:20:01.342 ZIG: timeout, goto label 99
00:20:01.350 MQT: tele/tasmota_E32AEB/RESULT = {"ZbState":{"Status":99,"Message":"Abort"}}
00:20:01.351 ZIG: Abort
00:20:01.352 ZIG: Stopping (99)
![image](https://user-images.githubusercontent.com/47158433/107712041-faaf8d00-6cc8-11eb-9fad-ede146899b4e.png)
![image](https://user-images.githubusercontent.com/47158433/107712105-1a46b580-6cc9-11eb-8366-0c564266b34c.png)
![image](https://user-images.githubusercontent.com/47158433/107712149-30ed0c80-6cc9-11eb-9594-7e2fdcb76caa.png)
![image](https://user-images.githubusercontent.com/47158433/107712261-68f44f80-6cc9-11eb-861e-dff36ff54f62.png)

Kind regards
Lucas

**Comment by Jason2866** [comment]
Maybe the USB Port does not deliver enough current for the device. Try a known to be good USB power source.

**Comment by Mordi1984** [comment]
Hi Jason. Thank you for the reply.
Yesterday i thought that too at first. I tried also an Other charger. First 1 Ampere, then 1,5 Ampere, and then 2 Ampere.
I disconnected the device for 6 hours, plugged it in again and it started again.
This evening i tried to switch on the light after 9 hours connected to the WiFi and it won´t start again.
Again Restart Zigbee not started.
I really don´t know why. The device came flashed. I reflashed it after stucked the first time.
And now, again.
Here is the LOG after reset from this evening:

19:20:52.040 MQT: tele/tasmota_E32AEB/INFO1 = {"Module":"Sonoff ZbBridge","Version":"9.2.0.6(zbbridge)","FallbackTopic":"cmnd/DVES_E32AEB_fb/","GroupTopic":"cmnd/tasmotas/"}
19:20:52.042 MQT: tele/tasmota_E32AEB/INFO2 = {"WebServerMode":"Admin","Hostname":"tasmota_E32AEB-2795","IPAddress":"192.168.178.131"}
19:20:52.045 MQT: tele/tasmota_E32AEB/INFO3 = {"RestartReason":"Software/System restart"}
19:20:55.442 QPC: Reset
19:20:56.412 MQT: tele/tasmota_E32AEB/STATE = {"Time":"2021-02-12T19:20:56","Uptime":"0T00:00:09","UptimeSec":9,"Vcc":3.476,"Heap":31.4,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":1,"Wifi":{"AP":1,"SSId":"Endlich Zuhause","BSSId":"C8:0E:14:A0:46:B1","Channel":11,"RSSI":100,"Signal":-32,"LinkCount":1,"Downtime":"0T00:00:03"}}
19:21:04.060 ZIG: Resetting EZSP device
19:21:05.335 MQT: tele/tasmota_E32AEB/RESULT = {"ZbState":{"Status":1,"Message":"EFR32 booted","RestartReason":"Power-on","Code":2}}
19:21:10.361 ZIG: timeout, goto label 99
19:21:10.366 MQT: tele/tasmota_E32AEB/RESULT = {"ZbState":{"Status":99,"Message":"Abort"}}
19:21:10.367 ZIG: Abort
19:21:10.368 ZIG: Stopping (99)

I really dont know if it is a Software problem or an hardware problem!

Kind regards
Lucas

Holen Sie sich Outlook für Android<https://aka.ms/ghei36>

**Comment by Jason2866** [comment]
Which EZSP firmware is flashed?

**Reply by Mordi1984** [reply]
Hi Jason,

Last SW is ncp-uart-sw_6.7.8_115200.ota
and the base is tasmota-zbbridge.bin

Is that the Info you need?

Kind regards, Lucas

**Comment by s-hadinger** [comment]
Can you provide logs with `Weblog 3`?

However this looks like a hardware problem.

**Reply by Mordi1984** [reply]
Hi, thank you for your Help.

This is the log:

00:00:00.032 CFG: Loaded from flash at F7, Count 8
00:00:00.035 QPC: Count 1
00:00:00.036 CFG: CR 360/699, Busy 0
00:00:00.038 ZIG: ZBBridge EEPROM found at address 0x50
00:00:00.039 SNS: Hardware Serial
00:00:00.040 SRC: Restart
00:00:00.041 Project tasmota Tasmota Version 9.2.0.6(zbbridge)-2_7_4_9(2021-02-06T17:20:13)
00:00:00.042 ZIG: Resetting EZSP device
00:00:00.192 WIF: Attempting connection...
00:00:00.528 WIF: Connecting to AP1 Endlich Zuhause Channel 11 BSSId C8:0E:14:A0:46:B1 in mode 11n as tasmota_E32AEB-2795...
00:00:01.752 WIF: Connected
00:00:02.002 HTP: Web server active on tasmota_E32AEB-2795 with IP address 192.168.178.131
00:00:02.513 RTC: UTC 2021-02-12T20:15:56, DST 2021-03-28T02:00:00, STD 2021-10-31T03:00:00
21:15:57.018 RSL: tele/tasmota_E32AEB/INFO1 = {"Module":"Sonoff ZbBridge","Version":"9.2.0.6(zbbridge)","FallbackTopic":"cmnd/DVES_E32AEB_fb/","GroupTopic":"cmnd/tasmotas/"}
21:15:57.019 RSL: tele/tasmota_E32AEB/INFO2 = {"WebServerMode":"Admin","Hostname":"tasmota_E32AEB-2795","IPAddress":"192.168.178.131"}
21:15:57.020 RSL: tele/tasmota_E32AEB/INFO3 = {"RestartReason":"Software/System restart"}
21:16:00.453 QPC: Reset
21:16:01.420 HTP: Main Menu
21:16:01.496 RSL: tele/tasmota_E32AEB/STATE = {"Time":"2021-02-12T21:16:01","Uptime":"0T00:00:09","UptimeSec":9,"Vcc":3.475,"Heap":30.6,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":0,"Wifi":{"AP":1,"SSId":"Endlich Zuhause","BSSId":"C8:0E:14:A0:46:B1","Channel":11,"RSSI":100,"Signal":-27,"LinkCount":1,"Downtime":"0T00:00:03"}}
21:16:02.412 APP: Boot Count 3
21:16:02.563 CFG: Saved to flash at 1FB, Count 9, Bytes 4096
21:16:03.822 HTP: Console
21:16:09.057 ZIG: Resetting EZSP device
21:16:10.327 RSL: tele/tasmota_E32AEB/RESULT = {"ZbState":{"Status":1,"Message":"EFR32 booted","RestartReason":"Power-on","Code":2}}
21:16:10.328 ZIG: ZbEZSPSend 000008
21:16:15.352 ZIG: timeout, goto label 99
21:16:15.354 RSL: tele/tasmota_E32AEB/RESULT = {"ZbState":{"Status":99,"Message":"Abort"}}
21:16:15.355 ZIG: Abort
21:16:15.355 ZIG: Stopping (99)
21:16:18.265 HTP: Main Menu
21:16:20.616 HTP: Console

Kind regards,
Lucas

**Comment by Barracuda09** [comment]
I have some fishy situation as well, that looks a bit like described here.
The main web page usually showed me the Zigbee devices connected. But not anymore, and it keeps on restarting with reason _exception_  as well no Zigbee map anymore and showing _Zigbee not started_. But the strange thing is that I can control the Zigbee devices by MQTT.


 . |  .
--|--
Program Version | 9.2.0.4(zbbridge)
Build Date & Time | 2021-02-01T16:48:15
Core/SDK Version | 2_7_4_9/2.2.2-dev(38a443e)

Edit resart reason send to MQTT:
`{"RestartReason":{"Exception":0,"Reason":"Exception","EPC":["4000dce5","00000000","00000000"],"EXCVADDR":"00000000","DEPC":"00000000","CallChain":["40286639","402300f4","4022fbbe","4024ea77","40100f3f","40243b70","40100f3f","40284eb9","4023a55e","4023ccc8","4023c7bb","4023cdb0","4021d670","4023029d","402193d9","402309f9","4021d670","40284f00","4023c9f4","40284f0e","4021d670","4021d670","4023177f","401011c7","40234fed","4023d03c","4021d670","40240916","40240974","401000e1","4021d6a8"]}}`

**Reply by Barracuda09** [reply]
Well I resolved my problem with removing the _friendlyname_ for most of the devices. Now it shows the devices again on the main page without restarting all the time.

Edit:
So it seems an configuration issue (bug) that will cause an _exception_ and render the device almost unusable when going to the web page.

**Reply by Mordi1984** [reply]
In my Case i haven´t choosen any friendly name and leaved them, and communicated with the stock given name.

**Reply by Barracuda09** [reply]
So in your case it starts OK with the first two power/restart cycles after re-flashing?

**Comment by Mordi1984** [comment, marked as answer]
Hmmm. Good question.
Last night i led it plugged out again.
In the morning after plugged in, it won't starts
Again status 99.
Then I reflashed first the lower version of the .*ota
Fith an foult, and then the upper version with an success.
Then it booted up and zigbee started again, with no fault.
I relearned my switch and could use it.
Iam afraight to restart it again because many times it reports after all status 99 again!
My learnefdevice shows off since 34 minutes.
Is there a suspend mode in the bridge? Or is it that it is switched off?

Kind regards
Lucas



Holen Sie sich Outlook für Android<https://aka.ms/ghei36>

________________________________

**Reply by s-hadinger** [reply]
No there is no suspend mode. From your past logs, it looks like the MCU is not responding at all. I'm still afraid of a hardware issue. I've never seen such things with my 2 ZBBridge and I restart them a lot.

**Reply by Barracuda09** [reply]
Well there are as well some software issues, see my comment https://github.com/arendst/Tasmota/discussions/10928#discussioncomment-364227

**Reply by s-hadinger** [reply]
It's possible that some weird friendly names cause issues.

**Reply by Barracuda09** [reply]
If weird is something like _Light_ or _TempHumi_, then yes.

**Reply by s-hadinger** [reply]
If you stick to ASCII then you're fine. There may be problems with spaces or Unicode

**Reply by Barracuda09** [reply]
Only ASCII chars and no spaces. Only these exact names.

**Comment by ea4aqm** [comment]
Hello, I solved the problem, I had the some problem. The hardware is fine and the (poor quality) USB cable does not cause problems.
Step 1: I have removed the friendlyname
Step 2: I have upgraded to "Sonoff Basic" by OTA url (http://ota.tasmota.com/tasmota/release/tasmota.bin.gz)
Step 3: I have upgraded to Sonoff ZbBridge using OTA URL (http://ota.tasmota.com/tasmota/release/tasmota-zigbee.bin.gz)
Step 4: I have updated by file Upload the file: ncp-uart-nsw_6.7.9_115200.ota
Step 5: now, It's working !!
I hope to help us.
Regards. Victor.

**Reply by aidux** [reply]
Hello, had the same issue. However, discovered that after fully disconnecting flashing device and all wires, uploading of FW file again goes normally. And after rebooting the issue disappeared.

**Comment by ludekvodicka** [comment]
Hi,
I have exactly the same issue. I tried several flashing / OTA updates /  fw updates, I tried it with/without connected wifi etc. Nothing helped.

Is there anything else I can try?

**Comment by ludekvodicka** [comment]
Based on what I read, I belive the issue is in DNS resolution of nl.pool.ntp.org. 

I'm not sure why but my tasmota doesn't resolve any DNS:

00:00:09.018 DNS: Unable to resolve 'pool.ntp.org'
00:00:09.223 CFG: Saved, Count 20, Bytes 4096
00:00:12.275 DNS: Unable to resolve 'nl.pool.ntp.org'
00:00:12.300 QPC: Reset
00:00:15.352 DNS: Unable to resolve '0.nl.pool.ntp.org'
00:00:15.357 HTP: Main Menu
00:00:18.460 DNS: Unable to resolve '0.pool.ntp.org'
00:00:18.474 ZIG: Resetting EZSP device
00:00:18.476 APP: Boot Count 5
00:00:21.528 DNS: Unable to resolve 'pool.ntp.org'
00:00:21.676 CFG: Saved, Count 21, Bytes 4096
00:00:24.726 DNS: Unable to resolve 'nl.pool.ntp.org'
00:00:25.033 RSL: RESULT = {"ZbState":{"Status":1,"Message":"EFR32 EZSP booted","RestartReason":"Power-on","Code":2}}
00:00:25.034 ZIG: ZbEZSPSend 000008
00:00:28.099 DNS: Unable to resolve '0.nl.pool.ntp.org'
00:00:31.162 DNS: Unable to resolve '0.pool.ntp.org'
00:00:31.180 ZIG: {"ZbEZSPReceived":"000008029067"}
00:00:31.181 RSL: RESULT = {"ZbState":{"Status":55,"Version":"6.7.9.0","Protocol":8,"Stack":2}}
00:00:31.193 RSL: RESULT = {"ZbState":{"Status":3,"Message":"Configured, starting coordinator"}}
00:00:31.193 ZIG: ZbEZSPSend 53001E0200
00:00:34.261 DNS: Unable to resolve 'pool.ntp.org'

I tried to configure DNS to 8.8.8.8 via IPAddress4 command, but this didn't help

**Reply by s-hadinger** [reply]
This has nothing to do with the issue above. If you have a dns problem please open a new discussion with detailed version information

**Comment by allansari** [comment]
Pls help , as i've the same issue ( i flashed almost three times) and it says :

 Verification of HEX file OK
 FLH: cc2652_flasher rx=23 tx=19 rst=15 bsl=22
 FLH: Flashing started (takes 5 minutes during which Tasmota is unresponsive)
 FLH: Flashing completed: OK
 FLH: Flash crc32 0x000000 - 0x2FFFF = bytes('1598929A')


but later i get this : 

00:00:00.002 HDW: ESP32-D0WD-V3 
00:00:00.058 UFS: FlashFS mounted with 720 kB free
00:00:00.075 CFG: Loaded from File, Count 27
00:00:00.082 QPC: Count 1
00:00:00.119 BRY: Berry initialized, RAM used=3522 bytes
00:00:00.134 Project tasmota - Tasmota Version 12.4.0.2(zig_bridge32)-2_0_7(2023-03-13T15:12:56)
00:00:00.135 ZIG: Resetting EZSP device
00:00:00.693 WIF: Connecting to AP1 IoT Channel 1 BSSId 20:0D:B1:8E:0B:B7 in mode 11n as tasmota-5F4E48-3656...
00:00:05.434 WIF: Connected
00:00:05.690 HTP: Web server active on tasmota-5F4E48-3656 with IP address 192.168.X.XXX
20:06:42.608 RSL: INFO1 = {"Info1":{"Module":"Sonoff Zigbee Pro","Version":"12.4.0.2(zig_bridge32)","FallbackTopic":"cmnd/DVES_5F4E48_fb/","GroupTopic":"cmnd/tasmotas/"}}
20:06:42.611 RSL: INFO2 = {"Info2":{"WebServerMode":"Admin","Hostname":"tasmota-5F4E48-3656","IPAddress":"192.168.1.228","IP6Global":"2001:1670:10:6bf4:4222:d8ff:fe5f:4e48","IP6Local":"fe80::4222:d8ff:fe5f:4e48"}}
20:06:42.613 RSL: INFO3 = {"Info3":{"RestartReason":"Software reset CPU","BootCount":12}}
20:06:42.863 QPC: Reset
20:06:46.912 RSL: STATE = {"Time":"2023-03-13T20:06:46","Uptime":"0T00:00:11","UptimeSec":11,"Heap":170,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":91,"MqttCount":0,"Berry":{"HeapUsed":3,"Objects":34},"Wifi":{"AP":1,"SSId":"IoT","BSSId":"20:0D:B1:8E:0B:B7","Channel":1,"Mode":"11n","RSSI":100,"Signal":-50,"LinkCount":1,"Downtime":"0T00:00:06"}}
20:06:51.557 ZIG: Resetting EZSP device
20:06:52.639 ZbEZSPReceived: time-out, discarding 00FE064180000201020701C0
20:06:56.753 ZIG: timeout, goto label 99
20:06:56.755 RSL: RESULT = {"ZbState":{"Status":99,"Message":"Abort"}}
20:06:56.756 ZIG: Abort
20:06:56.757 ZIG: Stopping (99)

**Reply by s-hadinger** [reply]
You claim to have flashed a cc2652 yet you are using an ezsp firmware. This is wrong. 

If it's really a cc2652, you should use a ZNP firmware

**Reply by allansari** [reply]
How to get ZNP ?

**Reply by s-hadinger** [reply]
Please check the Zigbee documentation

**Reply by allansari** [reply]
Oh man this like an assignment 😅

**Reply by allansari** [reply]
Pls help me , i don't know what do you mean by ZNP firmware. i used tasmota32-ZigBee bridge pro and followed all the steps and tool highlighted by @notenoughtech  : https://notenoughtech.com/home-automation/tasmota-on-sonoff-zb-bridge-pro/


here is the chip info:


Chip Info:
 - Chip Family: ESP32
 - Chip Model: ESP32-D0WDQ5-V3 (revision 3)
 - Number of Cores: 2
 - Max CPU Frequency: 240MHz
 - Has Bluetooth: YES
 - Has Embedded Flash: NO
 - Has Factory-Calibrated ADC: YES
 - MAC Address: 40:22:D8:5F:4E:48

**Comment by allansari** [comment]
i don't know what i've done, but it works finally - i made it :) 

the only thing i did is, flashed ZB with old version then i used the tasmota32 pro through OTA


00:00:00.002 HDW: ESP32-D0WD-V3 
00:00:00.125 UFS: FlashFS mounted with 292 kB free
00:00:00.147 CFG: Loaded from File, Count 33
00:00:00.157 QPC: Count 1
00:00:00.324 BRY: Berry initialized, RAM used=4041 bytes
00:00:00.340 TFS: File '.drvset003' not found
00:00:00.340 CFG: Energy use defaults as file system not ready or file not found
00:00:00.348 Project tasmota - Tasmota Version 12.4.0(zbbrdgpro)-2_0_6(2023-02-24T15:35:22)
00:00:01.038 WIF: Connecting to AP1 ASTROLABE-1 Channel 1 BSSId 20:0D:B0:8E:0D:AB in mode 11n as tasmota-5F4E48-3656...
00:00:04.859 WIF: Connected
18:28:11.094 HTP: Web server active on tasmota-5F4E48-3656 with IP address 192.168.X.XXX
18:28:11.100 UPP: Multicast (re)joined
18:28:12.111 RSL: INFO1 = {"Info1":{"Module":"Sonoff Zigbee Pro","Version":"12.4.0(zbbrdgpro)","FallbackTopic":"cmnd/DVES_5F4E48_fb/","GroupTopic":"cmnd/tasmotas/"}}
18:28:12.126 RSL: INFO2 = {"Info2":{"WebServerMode":"Admin","Hostname":"tasmota-5F4E48-3656","IPAddress":"192.168.1.228","IP6Global":"","IP6Local":"fe80::4222:d8ff:fe5f:4e48"}}
18:28:12.139 RSL: INFO3 = {"Info3":{"RestartReason":"Software reset CPU","BootCount":16}}
18:28:12.833 QPC: Reset
18:28:16.866 RSL: STATE = {"Time":"2023-03-14T18:28:16","Uptime":"0T00:00:11","UptimeSec":11,"Heap":165,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":37,"MqttCount":0,"Berry":{"HeapUsed":3,"Objects":47},"Wifi":{"AP":1,"SSId":"ASTROLABE-1","BSSId":"20:0D:B0:8E:0D:AB","Channel":1,"Mode":"11n","RSSI":56,"Signal":-72,"LinkCount":1,"Downtime":"0T00:00:05"}}
18:28:21.703 ZIG: rebooting ZNP device
18:28:22.623 ZbInput discarding byte 00
18:28:22.626 RSL: RESULT = {"ZbState":{"Status":1,"Message":"CCxxxx ZNP booted","RestartReason":"Power-up","MajorRel":2,"MinorRel":7}}
18:28:22.829 RSL: RESULT = {"ZbState":{"Status":50,"MajorRel":2,"MinorRel":7,"MaintRel":1,"Revision":20220219}}
18:28:22.880 RSL: RESULT = {"ZbState":{"Status":2,"Message":"Resetting configuration"}}
18:28:24.509 ZbInput discarding byte 00
18:28:25.074 RSL: RESULT = {"ZbState":{"Status":3,"Message":"Configured, starting coordinator"}}
18:28:30.990 RSL: RESULT = {"ZbState":{"Status":40,"NewState":9,"Message":"Started as coordinator"}}
18:28:31.092 RSL: RESULT = {"ZbState":{"Status":51,"IEEEAddr":"0x00124B00257BB224","ShortAddr":"0x0000","DeviceType":7,"DeviceState":9,"NumAssocDevices":0}}
18:28:31.602 RSL: RESULT = {"ZbState":{"Status":0,"Message":"Started"}}
18:28:31.605 ZIG: Zigbee started
18:28:31.614 ZIG: No Zigbee device information
18:28:31.619 ZIG: No Zigbee device data
18:28:31.620 ZIG: ZbLoad '<internal_plugin>' loaded successfully
18:28:38.662 RSL: RESULT = {"ZbState":{"Status":21,"Message":"Pairing mode enabled"}}
18:28:38.666 RSL: RESULT = {"ZbPermitJoin":"Done"}
18:28:47.859 ZIG: sending `ZbMap 0x0000`
18:28:47.924 RSL: RESULT = {"ZbMap":{"Device":"0x0000","Status":0,"StatusMessage":"SUCCESS","Total":0,"Start":1,"Map":[]}}
18:28:53.807 ZIG: ZbMap done
18:29:03.201 ZIG: sending `ZbMap 0x0000`
18:29:03.328 RSL: RESULT = {"ZbMap":{"Device":"0x0000","Status":0,"StatusMessage":"SUCCESS","Total":0,"Start":1,"Map":[]}}
18:29:09.208 ZIG: ZbMap done
18:29:14.825 RSL: RESULT = {"ZbState":{"Status":21,"Message":"Pairing mode enabled"}}
18:29:14.828 RSL: RESULT = {"ZbPermitJoin":"Done"}

**Reply by allansari** [reply]
Hi all

After doing all steps successfully, Unfortunately I couldn't pair the sonoff zbridge with Hubitat?

Any idea pls?

**Reply by s-hadinger** [reply]
Any logs? What device is this ?

**Reply by allansari** [reply]
It's "Sonoff Zigbee bridge Pro" based ESP32

**Reply by s-hadinger** [reply]
I know that. What is the device that you can't pair?

**Reply by allansari** [reply]
I want to convert the ZbridgePro as router and connect it to Hubitat Hub?!

**Reply by allansari** [reply]
> I know that. What is the device that you can't pair?

https://github.com/arendst/Tasmota/discussions/16957#discussioncomment-5313722
